docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -13,6 +13,151 @@ Release Notes
 Features
 ========
 
+New in Version 6.0.0
+--------------------
+This is a major release with a number of new features, enhancements, and bug fixes as described below. Many of the new
+features are now highlighted in our 'What's New' window displayed when SasView starts.
+
+This version of SasView is built with Sasmodels v1.0.7, Sasdata v0.9.0, and bumps v0.9.2.
+
+New Features
+^^^^^^^^^^^^
+* Improvements to Corfunc by @lucas-wilkins in #2450
+* Corfunc perspective needs export and report capabilities by @lucas-wilkins in #2065
+* Add a preferences window by @krzywon in #2167
+* Debye efficiency by @klytje in #2859
+* Rog and beta q by @smalex-z in #2535
+* Custom fit models by @smalex-z in #2565
+* 1763 implement the orientation viewer in 5x and GL Subsystem by @lucas-wilkins in #2394
+* Particle editor by @lucas-wilkins in #2520
```

Comment:
again, lines beginning with numbers that I don't understand? Probably just need to remove them?

ALSO ... I don't think the particle editor is actually complete and available is it?